### ISSUE TITLE
The issue of Spell ID = 58427（Rouge）

### ISSUE THREAD

**Issue by songyuyx** [rgd87/NugRunning#52]
Dear Sir,  I found a issue maybe a bug for the Spell Id=58427(Rouge) 

When sometime,   the timeline of 58427 is always keeping 100% without any reducion.  I attached a screenshoot to show it clear

![image](https://user-images.githubusercontent.com/55168335/188251784-d355bb2c-8445-49d8-9313-c4924520bf8d.png)

**Comment by d87** [comment]
You either got the wrong spell ID or you have  无限计时器 enabled
try _/nrun listauras player_

**Comment by songyuyx** [comment]
Dear Sir,  Really thanks replay to me.  I did the two test what you mention,  but ths issue is still keeping.  I attached the picture. 

Kindly notice,  the issue is happen **sometime**, not always.  maybe 50% probability。 
NugRunning is the best addon to buf delection for me, I use it since the Phase1 of TBC.  
I would like to more test to help fix the issue, pls no hesitate to let me know what needs to be test more

![image](https://user-images.githubusercontent.com/55168335/188532968-5a971b44-f941-410b-a1bb-56bae8060621.png)

![image](https://user-images.githubusercontent.com/55168335/188533216-b66eb411-c566-44f4-95c5-e9cabd68a605.png)
